### DISCUSSION TITLE
unibooty 0

### DISCUSSION THREAD

**Discussion by novaTopFlex** [nanovms/nanos#2013] [Q&A]
When "make run-noaccel" is running, I attempt to enter port 8080 in different IP addresses and am stuck on "unibooty 0" on my Chromium tab. How can I fix this?

**Comment by eyberg** [comment]
I'm not quite sure what you're asking. 'make run-noaccel' is merely a manual test and not what users use from day to day. It sounds like you want to use ops, https://ops.city and run 'ops run' or 'ops instance create'.

If on the other hand you are trying to develop against nanos itself we'd probably need a bit more information on what you are wanting to do here.
